soc/xtensa/intel_adsp/tools/cavstool_client.py

```diff
@@ -115,7 +115,20 @@ def main():
 if args.quiet:
     log.setLevel(logging.WARN)
 
-HOST = args.server_addr
+if args.server_addr:
+    h = args.server_addr.split(':')
+    l = len(h)
+
+    if l == 3:
+        PORT_LOG = int(h[2])
+        PORT_REQ = int(h[1])
+
+    elif l == 2 or l > 3:
+        log.error("Please use format --remote-host=HOST or HOST:REQ_PORT:LOG_PORT")
```

Comment:
Thanks for separating this from a previous, bigger and apparently not related PR. Copying my comment from the other PR:

> Please use a new --argument for this. Don't extend standard conventions; they're painful enough to parse already. The number of colons : is used as a trick to identify bracket-less IPv6 addresses which means {hostname}:{req_port}:{log_port} could be wrongly identified as an IPv6 address.

https://bugzilla.mindrot.org/show_bug.cgi?id=1602
https://chromium-review.googlesource.com/c/chromiumos/third_party/autotest/+/312719/3/server/base_utils.py

Reply:
Fixed it.